datadog/data_source_datadog_monitor.go

```diff
@@ -265,6 +265,11 @@ func dataSourceDatadogMonitor() *schema.Resource {
 					},
 				},
 			},
+			"notification_preset_name": {
+				Description: "Toggles the display of additional content sent in the monitor notification. Valid values are: `show_all`, `hide_query`, `hide_handles`, and `hide_all`.",
+				Type:        schema.TypeString,
+				Computed:    true,
```

Comment:
Should the datasource schema for this also have the enum validation function? There's one field in the monitor datasource that uses it (renotify_statuses) but since its read-only might not matter too much.

Reply:
Need to be explicit here. Terraform has safeguards for having `Validate` funcs in data sources (at least for primitive types but does seems to ignore it in `TypeSets`)